.github/workflows/swift-toolchain.yml

```diff
@@ -397,7 +405,8 @@ jobs:
 
   ds2:
     # TODO: Build this on macOS or make an equivalent Mac-only job
-    if: inputs.build_os == 'Windows'
+    # There is no Android NDK for Windows ARM64. So disable on Windows/ARM64.
```

Comment:
I'm not sure I understand this. What prevents us from shipping DS2 on Windows ARM64? We could use this for remote debugging Windows as well.

Reply:
I understand ds2 can be used on windows/arm64. But the ds2 job's matrix only contains Android ones (no Windows x64 nor arm64) and ds2_tools are only needed when the ds2 job executes. So I think this seems ok to me. Improved the comment a bit.